### PR TITLE
#46: Adding AppAwards, AppAnnouncements, AppAnnouncementsAwards compo…

### PR DESCRIPTION
**Pull request by serundeputy**
…nents.

### CODE REVIEW

File: components/AppAnnouncementsAwards/AppAnnouncements.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    };
+  }
+};
+</script>
+<style scoped>
+.announcement {
+  padding: 9px;
+  border-bottom: 1px var(--gray-200) solid;
+}
+.announcement-date {
```
Comment (dustinleblanc) [inline components/AppAnnouncementsAwards/AppAnnouncements.vue]: Lets change this class to `.announcement__date` to folllow BEM

File: components/AppAnnouncementsAwards/AppAnnouncementsAwards.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+export default {
+  components: {
+    AppAnnouncements,
+    AppAwards
+  }
+};
+</script>
+<style scoped>
+.promo__disclosure {
```
Comment (dustinleblanc) [inline components/AppAnnouncementsAwards/AppAnnouncementsAwards.vue]: Do we have a promo in this component section?

File: components/AppAnnouncementsAwards/AppAwards.vue [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="bg-white p-4">
+    <h2>Awards</h2>
+    <p
+      v-for="award in awards"
+      :key="award.img">
+      <img
```
Comment (dustinleblanc) [inline components/AppAnnouncementsAwards/AppAwards.vue]: let's use a `<b-img>` tag here so we can take advantage of the [bootstrap stuffs](https://bootstrap-vue.js.org/docs/components/image). I'd recommend that we set some static sizes and throw in some blanks here to indicate the size/shape/placement

**Comment by serundeputy** [comment]
Thanks @dustinleblanc those comments make sense to me; I've addressed them ... I'm going to merge this so I can iterate on the other components and we can always keep making bootstrap and BEM improvements as we go.

**Pull request merged**